validphys2/src/validphys/pseudodata.py

```diff
@@ -80,25 +80,40 @@ def read_replica_pseudodata(fit, context_index, replica):
     log.debug(f"Reading pseudodata & training/validation splits from {fit.name}.")
     replica_path = fit.path / "nnfit" / f"replica_{replica}"
 
-    training_path = replica_path / (FILE_PREFIX + "training_pseudodata.csv")
-    validation_path = replica_path / (FILE_PREFIX + "validation_pseudodata.csv")
-
     try:
+        # The paths where sampled experimental data is stored
+        training_path = replica_path / "datacuts_theory_fitting_training_pseudodata.csv"
+        validation_path = replica_path / "datacuts_theory_fitting_validation_pseudodata.csv"
         tr = pd.read_csv(training_path, index_col=[0, 1, 2], sep="\t", header=0)
         val = pd.read_csv(validation_path, index_col=[0, 1, 2], sep="\t", header=0)
     except FileNotFoundError:
-        # Old 3.1 style fits had pseudodata called training.dat and validation.dat
-        training_path = replica_path / "training.dat"
-        validation_path = replica_path / "validation.dat"
-        tr = pd.read_csv(training_path, index_col=[0, 1, 2], sep="\t", names=[f"replica {replica}"])
-        val = pd.read_csv(
-            validation_path, index_col=[0, 1, 2], sep="\t", names=[f"replica {replica}"]
-        )
-    except FileNotFoundError as e:
-        raise FileNotFoundError(
-            "Could not find saved training and validation data files. "
-            f"Please ensure {fit} was generated with the savepseudodata flag set to true"
-        ) from e
+        try:
+            # Pseudodata generated in a closure test
+            training_path = (
+                replica_path / "datacuts_theory_closuretest_fitting_training_pseudodata.csv"
+            )
+            validation_path = (
+                replica_path / "datacuts_theory_closuretest_fitting_validation_pseudodata.csv"
+            )
+            tr = pd.read_csv(training_path, index_col=[0, 1, 2], sep="\t", header=0)
+            val = pd.read_csv(validation_path, index_col=[0, 1, 2], sep="\t", header=0)
+        except FileNotFoundError:
+            try:
+                # Old 3.1 style fits had pseudodata called training.dat and validation.dat
+                training_path = replica_path / "training.dat"
+                validation_path = replica_path / "validation.dat"
+                tr = pd.read_csv(
+                    training_path, index_col=[0, 1, 2], sep="\t", names=[f"replica {replica}"]
+                )
+                val = pd.read_csv(
+                    validation_path, index_col=[0, 1, 2], sep="\t", names=[f"replica {replica}"]
+                )
```

Comment:
So, I personally would go for this:

```python
trdata = Path(.../"datacuts_theory_fitting_training_pseudodata.csv")
trclosure = Path(.../"datacuts_theory_closuretest_fitting_training_pseudodata.csv")
tr31 = Path(../<3.1 version>)

if trdata.exists() and trclosure.exists():
    raise SomeError("Found both closure and normal data, please make up your mind and remove one of the two")
elif trdata.exists():
    tr = pd.read_csv
elif trclosure.exist():
    tr = pd.read_csv
elif tr1.exist(): # We reach 3.1 only as a fallback.
    tr = pd.read_csv
else:
    raise FileNotFound
```

Reply:
Actually, do we still want to check for the 3.1 version? It has bee a long time since I last looked at such a fit, and this function is not called in any of the standard scripts for as far as I'm aware.

Reply:
If you want to remove it I certainly won't stop you.

Reply:
> So, I personally would go for this:

I think both approaches work fine and it's a matter of personal taste. I would go for @scarlehoff's implementation as well, since is more readable and coincise.

RE 3.1, I honestly don't know. Is there a case in which we might want to use 3.1 fits?